Find a functionally equivalent code:android.ddm.DdmHandleProfiling.handleMPSS:COMMENT
Method Modifier: private     
Comment:/*
     * Handle a "Method Profiling w/Streaming Start" request.
     */

@@ -2,8 +2,8 @@
     ByteBuffer in = wrapChunk(request);
     int bufferSize = in.getInt();
     int flags = in.getInt();
-    if (false) {
+    if (DEBUG) {
         Log.v("ddm-heap", "Method prof stream start: size=" + bufferSize + ", flags=" + flags);
     }
     try {

